service/awsconfig/v7/resource/cloudformation/create_test.go

```diff
@@ -96,25 +97,26 @@ func Test_Resource_Cloudformation_newCreate(t *testing.T) {
 			IAM:            &adapter.IAMClientMock{},
 		}
 		c.Logger = microloggertest.New()
+		c.Service = &cloudformationservice.CloudFormation{}
 
 		newResource, err = New(c)
 		if err != nil {
-			t.Error("expected", nil, "got", err)
+			t.Fatal("expected", nil, "got", err)
```

Comment:
I changed this because I saw all tests panicing and I had no idea why. With this the tests fail and tell you where and what is going on.